bin/prof_spacetime.ml

```diff
@@ -11,6 +11,9 @@ type command =
         print_symbol:      bool;
         print_line_number: bool; }
   | Process
+  | Diff of
```

Comment:
This command should be able to run on either processed or unprocessed profiles, based on the `-p` option, just like the other commands. So I would expect to see a `processed` field here like in the other cases.